photon-ml/src/main/scala/com/linkedin/photon/ml/util/VectorUtils.scala

```diff
@@ -16,6 +16,7 @@ package com.linkedin.photon.ml.util
 
 
 import breeze.linalg.{DenseVector, SparseVector, Vector}
+import org.apache.spark.mllib.linalg.{DenseVector => SDV, SparseVector => SSV, Vector => SV}
```

Comment:
"SV"? In casual reading without seeing this import line, I would have thought it meant "SparseVector" already.